dist/AppImageBuilder.yml

```diff
@@ -2,9 +2,11 @@ version: 1
 
 script:
 - |
-  icon_path="{{APPIMAGE_PATH}}/usr/share/icons/hicolor/128x128/apps"
+  cp -rl "{{APPIMAGE_PATH}}" "${TARGET_APPDIR}"
+  icon_path="${TARGET_APPDIR}/usr/share/icons/hicolor/128x128/apps"
   mkdir -p "${icon_path}"
   cp -a dist/evdevhook2.png "${icon_path}"
+  apt-get update && apt-get install -y --no-install-recommends squashfs-tools
```

Comment:
I think a comment linking to https://github.com/AppImageCrafters/build-appimage/issues/5 for future reference would be nice.

Reply:
In the code or in the git commit message?

Reply:
I added a comment in the code.